src/language/jump_zh_CN.properties

```diff
@@ -2114,10 +2114,10 @@ ui.plugin.OpenProjectPlugIn.open-project = 打开项目
 ui.plugin.OptionsPlugIn.selection-style = 选择风格
 ui.plugin.OptionsPlugIn.snap-vertices-tools = 顶点对齐工具
 ui.plugin.OptionsPlugIn.view-edit = 查看/编辑
-ui.plugin.PasteItemsPlugIn.clipboard-must-not-be-empty = \#T\:
-ui.plugin.PasteItemsPlugIn.clipboard-must-contain-geometries-or-wkt = \#T\:
-ui.plugin.PasteLayersPlugIn.clipboard-must-not-be-empty = \#T\:
-ui.plugin.PasteLayersPlugIn.clipboard-must-contain-layers = \#T\:
+ui.plugin.PasteItemsPlugIn.clipboard-must-not-be-empty=#T:
+ui.plugin.PasteItemsPlugIn.clipboard-must-contain-geometries-or-wkt=#T:
+ui.plugin.PasteLayersPlugIn.clipboard-must-not-be-empty=#T:
+ui.plugin.PasteLayersPlugIn.clipboard-must-contain-layers=#T:
```

Comment:
unrelated. how about we remove translations that were not maintained for years.

i expect the quality of those to be underwhelming at least.